src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -453,33 +453,37 @@ static int add_boot(const char *what) {
         _cleanup_blkid_free_probe_ blkid_probe b = NULL;
         const char *fstype = NULL, *uuid = NULL;
         sd_id128_t id, type_id;
+        const char *esp;
         int r;
 
         assert(what);
 
         if (!is_efi_boot()) {
```

Comment:
ah, indeed. you are right, we should just mount it always, if it is there. Will add another patch to this PR to drop this check.